contributing/dbt-core-docs-release-checklist

```diff
@@ -0,0 +1,20 @@
+Here are the details of the dbt Core docs release in checklist form. We can copy this content to each new docs release issue, using the stage of the release.
+
+## Beta release
+
+- [ ] Use consistent banner text. The following is appropriate for both beta and RC:
+      > You are currently viewing v1.1, which is a prerelease of dbt Core. The latest stable version is v1.0
+
+- [ ] Change the install path to the correct version
+- [ ] Update page notes from `beta` to `rc`.
+- [ ] Add blurb about changes to the migration guide about v1.1-specific content/changes
+- [ ] Merge PR once the beta has released.
+ 
+ ## RC release
+- [ ]  
+ 
+ 
+ ## GA release
+- [ ] Remove the prerelease banner:
+      > You are currently viewing v1.1, which is a prerelease of dbt Core. The latest stable version is v1.0
+- [ ] Make sure all version-specific docs changes are reflected in the migration guide. This doc should serve as a high-level overview of what's changed, more easily digestible than the GitHub repo changelog. You can use a [GitHub search](https://github.com/dbt-labs/docs.getdbt.com/pulls?q=is%3Apr+is%3Aclosed+label%3A%22dbt-core+v1.1.x%22) to find applicable changes.
```

Comment:
```suggestion
- [ ] Make sure all version-specific docs changes are reflected in the migration guide. This doc should serve as a high-level overview of what's changed, more easily digestible than the GitHub repo changelog. You can use a [GitHub search](https://github.com/dbt-labs/docs.getdbt.com/pulls?q=is%3Apr+is%3Aclosed+label%3A%22dbt-core+v1.1.x%22) to find applicable changes.
- [ ] Remove the "release candidate" `:::info` callout
- [ ] Update changelog link to point to release branch: https://github.com/dbt-labs/dbt-core/blob/1.1.latest/CHANGELOG.md (1.1.latest instead of HEAD)
```